src/jit-format/jit-format.cs

```diff
@@ -464,7 +464,15 @@ public static int Main(string[] args)
             {
                 if (!RunClangFormat(filenames, config.Fix, verbose))
                 {
-                    Console.WriteLine("Clang-Format needs to be rerun in fix mode");
+                    Console.WriteLine("Clang-format found formatting errors. Run:");
+                    if (returncode == -1)
```

Comment:
You'd never run with --noformat when you run --fix with clang-tidy, as clang-tidy doesn't correctly format the code it changes. So the options are --fix or --fix --untidy.